src/hearings/store/reducers/hearing-actuals.reducer.ts

```diff
@@ -31,6 +31,12 @@ export function hearingActualsReducer(
         }
       };
     }
+    case fromActions.SUBMIT_HEARING_ACTUALS_SUCCESS: {
+      console.log('CURRENT STATE', currentState);
```

Comment:
could this console.log be removed?

Reply:
After the submission is successful, it should clear the state(like set with initialHearingActualsState) rather than keep the current state, otherwise when you get into the journey again it will keep everthing the same for last e2e journey.

Reply:
I have updated the code as you suggested.